extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -132,7 +132,7 @@ export class AssessmentSummaryCard implements vscode.Disposable {
 
 		const assessmentResultPreText = view.modelBuilder.text().withProps({
 			value: constants.ASSESSMENT_RESULTS.toLocaleUpperCase(),
-			description: "", //TODO - add description later
+			description: constants.ASSESSMENT_RESULTS.toLocaleUpperCase(), //TODO - add description later
```

Comment:
remove comment if required description is added.

Reply:
I would suggest changing the constants.ASSESSMENT_RESULTS to an uppercase value instead of using a function to do it.

Reply:
@Ramudaykumar the description is not there in mocks, so I gave a sample description for now. And added TODO to update it once I get it.

@brian-harris constants.ASSESSMENT_RESULTS is used at other place as well where it is not in uppercase. Other way would be to create duplicate upper case string.

Reply:
I try not to share resource strings in code, unless I am confident the context is the same and when changing the string it won't break somewhere else that is being used.  
I suggest a new string resoure.

Reply:
I made separate string resources.